sources/be.json

```diff
@@ -1,7 +1,8 @@
+
 [
     {
-        "url": null,
-        "adapter": "eea",
+        "url": "http://discomap.eea.europa.eu/map/fme/latest/BE_<pollutant>.csv",
```

Comment:
I sorta want this to be 

```
"url": ""http://discomap.eea.europa.eu/map/fme/latest/"
```

and have the rest taken care of in the adapter, but I don't know why. I'm fine with this if you like it this way.